tutorial_sarscov2.md

```diff
@@ -0,0 +1,231 @@
+
+conda create -n V-pipe -c bioconda snakemake conda
+conda activate V-pipe
+# We will let snakemake --use-conda handle
+# installation and download of V-pipe dependencies
+```
+> **Tips:**  To save space, you can install the *-minimal* version of snakemake, without all the GUI dependencies
+
+## Clone V-pipe
+
+
+```bash
+git clone -b sars-cov2 https://github.com/cbg-ethz/V-pipe.git
+cd V-pipe
+```
+> **Versions**: different version come with different defaults
+> - *sars-cov2* branch is adapted for SARS-CoV-2
+> - the default branch (*master*) is adapted for HIV
+
+## Test V-pipe
+
+Put the `samples` hierarchy that you created before in this `V-pipe` directory. Then verify that it has the [desired structure](#preparing-a-small-dataset) :
+
+```bash
+tree samples
+```
+(or alternatively: `find samples`[^fancyfind])
+
+[^fancyfind]: `find samples | sed -e "s/[^-][^\/]*\// |/g" -e "s/|\([^ ]\)/|-\1/"` is overly long but produces slightly prettier output than `find samples`
+
+Check the parameters of the `vpipe.config` file with you favourite editor (`vim`, `emacs`, `nano`, [butterflies](https://xkcd.com/378/), etc.).
+For a SNV and Local (windowed) haplotype reconstruction, you will need at least the following options:
+
+```
+[input]
+reference = references/NC_045512.2.fasta
+
+[output]
+snv = True
+local = True
+global = False
+
+[general]
+aligner = bwa
+```
+
+Check what will be executed:
+```bash
+snakemake -s vpipe.snake --dryrun -p --cores 2
+```
+
+As it is your first run of V-pipe, this will also generate the sample collection table.
+Check `samples.tsv` in your editor.
+
+Note that the demo files you downloaded have only have reads of length 150.
+V-pipe's defaults parameters are optimized for reads length of 200 ; add the third column in the file:
+
+```
+SRR10903401	20200102	150
+SRR10903402	20200102	150
+```
+
+> **Tips:** Always check the content of the `samples.tsv` file.
+>  - If you didn't use the [correct structure](#preparing-a-small-dataset) this file might end up empty or some entries might be missing.
+>  - You can safely delete it and re-run the `--dryrun` to regenerate it
+
+Run the V-pipe analysis (the necessary dependencies will be downloaded and installed in conda environments managed by snakemake):
+
+```bash
+snakemake -s vpipe.snake --use-conda -p --cores 2
+```
+
+
+## Larger dataset
+
+Log onto [Euler](https://scicomp.ethz.ch/wiki/Euler) with ETH account.
+
+```bash
+ssh login.euler.ethz.ch
+```
+
+There is a much larger test data that you can use:
+
+```bash
+ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/
+ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/PRJNA6*
+ls /cluster/work/bewi/members/mpirkl/Sars-Cov-2_data/PRJNA6*/sample*/
+```
+
+## Cluster deployment
+
+It is possible to ask snakemake [submit jobs on a cluster](https://snakemake.readthedocs.io/en/stable/executing/cluster-cloud.html#cluster-execution), such as [LSF system used on Euler](https://scicomp.ethz.ch/wiki/Using_the_batch_system)
```

Comment:
It is possible to run V-pipe on a cluster with a job scheduling system, such as LSF

Reply:
I wanted to link to the snakemake documentation explaining how to do it.